### DISCUSSION TITLE
AI for ESG！以No-Code平台降低AI導入門檻

### DISCUSSION THREAD

**Discussion by 32844583** [1101-GS3073-C/Python-Community-GS3073-C#520] [General]
### 用 No-Code AI 為產業賦能
近兩年來，台灣製造業對於如何將人工智慧有效地實施在各種場景中，已經有了一定的了解和建立了標準作業程序（SOP）。為了降低企業導入AI的門檻，詠鋐智能建立了一個有效的介面操作方式，透過No-Code AI平台的建立，讓現場人員可以簡單地使用，「自己的製程自己管」。目前的No-Code或是Low-Code平台可以分成兩大類，一類為雲端大廠提供的服務，主要是提供給開發商使用，需要具備資料科學的相關知識；另一類則是針對特定場域與產業所開發出的工具，後者更適合商業用戶，詠鋐智能的平台就屬於這一類。
https://www.digitimes.com.tw/iot/article.asp?id=0000643422_QZR3BMWP3NCRCI1YR1G62

**Comment by NcuJerryKao** [comment]
No-Code AI 感覺就算我這樣的 也能參與討論了

**Comment by williamyen0623** [comment]
No-Code AI真的的大大降低企業使用AI的門檻，讓非科技產業也能順利接軌人工智慧發展

**Comment by Ben6uo** [comment]
但感覺反而對CS需求下降 不知道會不會有反彈
